integration_tests/global_syms_03_a.py

```diff
@@ -4,4 +4,8 @@
 
 def populate_lists() -> list[i32]:
     return [10, -20]
-l_2: list[i32] = populate_lists()
+l_2: list[i32]
+l_2 = populate_lists()
```

Comment:
Why this change?

Reply:
I wanted to add an assignment statement for the initializer. Instead of adding a new test, I just used the existing one 😅.

Reply:
I would recommend to have both tests, initialiser and assignment statement for the initialiser.

Reply:
Got it! I will update the tests.